app/controllers/downvotes_controller.rb

```diff
@@ -0,0 +1,29 @@
+class DownvotesController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @vote = current_user.votes.new
+    @review = Review.find(params[:review_id])
+    sitcom = @review.sitcom
+    @vote.review_id = params[:review_id]
+
+    unless current_user.votes.where(review_id: @review.id).length > 1
+      user_vote = current_user.user_vote_by_review(@review)
+
+      if user_vote == nil
+        @vote.first_downvote
+      elsif user_vote.is_downvote?
+        user_vote.to_neutral
+      elsif user_vote.is_neutral? || user_vote.is_upvote?
+        user_vote.to_downvote
+      else
+        flash[:notice] = "Something Went VERY wrong"
```

Comment:
Do you have any tests that assert this flash ever comes up? More specifically I'm not sure that the flash is persisted when you do a redirect. See if you can verify if this flash ever comes up through any interaction with the system?